sponsor.html

```diff
@@ -106,7 +106,113 @@
+            model, we are expanding to the west coast, creating a new venue location 
+            in San Francisco in order to increase our in-person event capacity and 
+            enable west coast hackers to participate.
+          </p>
+          <p>
+            Technica is not a typical hackathon—it serves as a place for the brightest 
+            thinkers in the country to come together to collaborate and share their 
+            innovative ideas. Our focus is on providing a welcoming, engaging, 
+            and creative environment to support <b>women and non-binary people</b> in tech 
+            culture. We pride ourselves in helping both beginners and experienced 
+            hackers explore technology and develop their skills. 
+          </p>
+          <p>
+            Not only does Technica serve as an incubator for creating and promoting 
+            a gender-diverse workforce in the future, but it also provides an optimal 
+            opportunity for recruitment and outreach. We would love for you to get 
+            involved as a sponsor for Technica 2021! 
+          </p>
+          <p>
+            Our available sponsorship tiers are listed on the following page. Additionally, 
+            we are happy to state that Technica has <b>501(c)(3) status.</b> Please let 
+            us know if you are interested in becoming a part of Technica, or if 
+            you have any questions or concerns about the information below. We look 
+            forward to hearing from you!
+          </p>
+          <p>
+            Thank you,
+            <br>
+            The Technica Team
+          </p>
+        </div>
+      </section>
+      
+    </article>
+  </div>
+
+  <!-- stats -->
+  <div class = "stats">
+    <div class="overall-stats">
+      <div class="heading">
+        Statistics
+      </div>
+      <div class="row">
+        <div class="col-lg-5">
+          <div class="row" style="margin-bottom: 30px">
+            <div class="col-6">
+              <span class="stat-number">850+</span>
+              <br>
+              <span class="stat-text">Attendees</span>
+            </div>
+            <div class="col-6">
+              <span class="stat-number">150+</span>
+              <br>
+              <span class="stat-text">Schools</span>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-6">
+              <span class="stat-number">144</span>
+              <br>
+              <span class="stat-text">Projects submitted</span>
+              </div>
+            <div class="col-6">
+              <span class="stat-number">37</span>
+              <br>
+              <span class="stat-text">Sponsors</span>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-7 pl-0">
+          <img src="./img/statistics.jpg" alt="A group photo of a team of 4 hackers sitting in a lecture hall, situated near the end of the aisle.">
```

Comment:
couple notes on best practices for images:
1. the file name should reflect what it is (in this case, winning-hackers would work better)
2. the `alt` tag is a description of the image for the screen reader to relay to users who can't see the image (in this case, A group photo of 5 hackers standing and smiling on the awards ceremony stage. would work better)

Reply:
can you make this image smaller, so that the bottom aligns with the second row of stats (see figma for ref)